src/components/news-ticker/news-ticker.scss

```diff
@@ -144,7 +144,7 @@ $news-ticker: null !default;
     background: map.get($news-ticker, 'container-background');
     border-inline-end: map.get($news-ticker, 'border');
     height: map.get($news-ticker, 'desktop-only-height');
-    padding: var(--s-s) var(--s-6xl) var(--s-s) var(--s-m);
+    padding: map.get($news-ticker, 'container-padding-desktop');
```

Comment:
funny, looking at the demo it seemed to work, but this should be `padding-container-desktop` based on the name of the key in the maps. :)